### DISCUSSION TITLE
"Better Or Nothing" mod

### DISCUSSION THREAD

**Discussion by Ransomwave** [ppy/osu#27544] [Ideas]
This would be a mod that would function similarly to Sudden Death, but it would only kill you if you did worse score/combo than your previous best. It could also be configurable for you to set a score/combo threshold you want to hit it kills you.

Me and many other people would this very useful as I personally find myself constantly retrying beatmaps to get better scores than before, so that would save me the hustle of hitting esc and retry each and every time.

**Comment by EVAST9919** [comment]
Similar to https://github.com/ppy/osu/discussions/13069
